data/korath/korath jobs.txt

```diff
@@ -243,3 +243,588 @@ mission "Kor Efreti to Wanderers: Farming"
 	on complete
 		payment 10000 200
 		dialog `When you return, the Korath move their hands in a circular motion with their palms toward you as a sign of gratitude. They hand you <payment> and disembark your ship.`
+
+mission "Korath Humanika: Fish Trade (small)"
+	name "New Tortuga fish trade"
+	job
+	repeat
+	description "Take <tons> of fish to <stopovers> and return with <tons> of supplies for <payment>. Avoid getting caught or you may be fined."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 5 10 0.8
+	illegal 50000
+	deadline 7
+	source "New Tortuga"
+	destination "New Tortuga"
+	stopover "La'ab Bamen"
+	to offer
+		has "event: know about Korath fish jobs"
+		not "Korath Humanika: Fish Trade (small): active"
+	on stopover
+		dialog "You drop of your <tons> of illicit fish, and receive <tons> of unknown cargo from a woman in a blue dress."
+	on complete
+		payment 25000 500
+		dialog "A woman in a blue dress thanks you for the delivery and pays you <payment>."
+		"reputation: Korath" ++
+
+mission "Korath Humanika: Fish Trade (medium)"
+	name "New Tortuga fish trade"
+	job
+	repeat
+	description "Take <tons> of fish to <stopovers> and return with <tons> of supplies for <payment>. Avoid getting caught or you may be fined."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 15 50 0.8
+	illegal 100000
+	deadline 7
+	source "New Tortuga"
+	destination "New Tortuga"
+	stopover "La'ab Bamen"
+	to offer
+		random < 20
+		has "event: know about Korath fish jobs"
+		not "Korath Humanika: Fish Trade (medium): active"
+		"cargo space" > 20
+	on stopover
+		dialog "You drop of your <tons> of illicit fish, and receive <tons> of unknown cargo from a woman in a blue dress."
+	on complete
+		payment 35000 400
+		dialog "A woman in a blue dress thanks you for the delivery and pays you <payment>."
+		"reputation: Korath" += 2
```

Comment:
Like I mentioned on the Discord server, this is probably worth pairing with reputation limits/modifiers once they're merged in. Either through a simple reputation cap, or by using the gain/loss modifiers I'll implementing in a followup PR. That can make reputation loss hit much harder if you're on the positive end, so capturing a worldship would obliterate most reputation gains that people would make.

Reply:
The penalty for capturing a Korath ship is intentionally much higher now.

Reply:
Given the rather massive populations on Korath ships, I don't think we need to worry about having a cap on how much reputation one can earn with these jobs. These aren't big reputation boosts, and not jobs that can be done in huge quantities very easily, either.

Reply:
When I proposed it, @ravenshining wasn't thrilled about my plan to have jobs that make the Korath like you, but she seemed more willing to accept single-digit numbers. I think double-digit numbers might be okay, but it can't go high enough where a few jobs will make the Korath like you again after capturing a few ships.

Reply:
Single digit is reasonable.

Reply:
I have approximately doubled the rep boosts. The largest jobs are slightly more than doubled, so it is feasible to repay disabling and pillaging ships if the player does a *lot* of missions. Capturing Korath ships has a huge penalty now, so it isn't really feasible to repay that reputation debt.